environments/production/app-processes.yml

```diff
@@ -68,14 +68,14 @@ celery_processes:
       concurrency: 50
     saved_exports_queue:
       num_workers: 3
-      concurrency: 2
+      concurrency: 4
```

Comment:
i would usually vote for increasing the workers just because i find that more effective w/o any supporting facts.
What's your quick thought on increasing workers vs increasing concurrency here?
note: i assume that for bigger saved exports the process would consume higher memory.

Reply:
There was some chat about this in #celery.  The conclusion seemed to be that increasing concurrency provides the least memory overhead.  It does seem possible that it would have less redundancy, though, if a child process issue brings down a worker.